executor/hnswpsql.py

```diff
@@ -13,6 +15,8 @@
 from .hnswlib_searcher import HnswlibSearcher, DEFAULT_METRIC
 from .postgres_indexer import PostgreSQLStorage
 
+STOP_SYNC_THREAD = False
```

Comment:
do not use a global, use a `threading.Event` or a `flag` as attribute to communicate between threads